ln-resource-mgr/src/lib.rs

```diff
@@ -138,6 +138,8 @@ pub enum FailureReason {
     NoResources,
     /// The outgoing peer has insufficient reputation for the htlc to occupy protected resources.
     NoReputation,
+    /// Upgradable signal has been modified.
```

Comment:
nit: note that this means the htlc has been tampered with and should be failed back.